docs/examples/Hybrid_Search.ipynb

```diff
@@ -40,16 +40,19 @@
   },
```

Comment:
<p>There is a KeyError here? I'll look into this — please hold off on merging till we fix this please</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qdrant/fastembed/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>IIRC, I just did not execute the cell with embedding computation, sorry</p><p> </p>

Reply:
I'll update the output

Reply:
I've updated the output now.